src/Migration/Migration1607529255AdyenNotification.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types=1);
+
+namespace Adyen\Shopware\Migration;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+class Migration1607529255AdyenNotification extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1607529255;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->executeUpdate("
+            ALTER TABLE `adyen_notification`
+            ADD COLUMN `scheduled_processing_time` DATETIME(3) NULL AFTER `processing` COMMENT 'Scheduled for';
+        ");
+    }
+
+    public function updateDestructive(Connection $connection): void
+    {
```

Comment:
Add a drop column for the destructive update.

Reply:
I read about migrations nowadays and I am not sure about it but I think the updateDestructive is not for the downgrading but for the commands that cannot be reversed.
https://docs.shopware.com/en/shopware-platform-dev-en/developer-guide/migrations#modifying-the-database

Let's try to validate it with SW6 and then maybe we need to change the previous migration scripts.

Reply:
Oh! That actually makes a lot of sense! :)